gensim/similarity_learning/custom_losses.py

```diff
@@ -0,0 +1,10 @@
+from keras import backend as K
+from keras.layers import Lambda
+
+
+def rank_hinge_loss(y_true, y_pred):
+    margin = 1
+    y_pos = Lambda(lambda a: a[::2, :], output_shape=(1,))(y_pred)

```

Comment:
Separate the lambda functions into function calls, if need to use similar functionality again